meshtastic/mesh_interface.py

```
@@ -2359,30 +2379,38 @@ def _send_packet(
+                    node_num = node.get("num") if isinstance(node, dict) else None
+                if node_found:
+                    if isinstance(node_num, int):
+                        nodeNum = node_num
+                    else:
+                        raise MeshInterface.MeshInterfaceError(
+                            _format_missing_node_num_error(destinationId)
+                        )
+                elif has_nodes:
+                    raise MeshInterface.MeshInterfaceError(
+                        f"NodeId {destinationId} not found in DB"
+                    )
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)